src/builder.rs

```diff
@@ -1406,6 +1417,31 @@ mod tests {
         );
     }
 
+    #[test]
+    fn descriptions() {
+        let thing = ThingBuilder::new("MyLampThing")
+            .description("My Lamp")
+            .descriptions(|ml| ml.add("en", "My lamp").add("it", "La mia lampada"))
+            .build()
+            .unwrap();
+
+        assert_eq!(
+            thing,
+            Thing {
+                context: TD_CONTEXT.into(),
+                title: "MyLampThing".to_string(),
+                description: Some("My Lamp".to_string()),
+                descriptions: Some(
+                    [("en", "My lamp"), ("it", "La mia lampada")]
+                        .into_iter()
+                        .map(|(k, v)| (k.to_string(), v.to_string()))
+                        .collect()
+                ),
+                ..Thing::empty()
+            }
+        );
+    }
+
```

Comment:
Following my previous point, we should "just" test the info is set accordingly to what has been passed to the builder.